client/src/components/Post/Post.tsx

```diff
@@ -248,12 +299,46 @@ class Post extends Component<Props, State> {
                     <div className={'post-comments-container'}>
                         {
                             this.state.comments.map((comment, index) => (
-                                <div key={index} className={'comment-card'}>
+                                <div key={index} className={'comment-card'}
+                                     onMouseEnter={() => this.setIsShown(index)}
+                                     onMouseLeave={() => this.setIsShown(-1)}>
                                     <div className={'comment-header'}>
                                         {comment.username + ' - ' + comment.creationDate + ' '}
```

Comment:
@Agusdelavega missed it last week: can we make this date + time more "human readable", in the likes of "x minutes/hours/days ago"
You can use a react library to help you with it, for example:
https://www.npmjs.com/package/react-time-ago